ansible-playbook-base/site.yml

```diff
@@ -64,3 +64,23 @@
     - create-gluster-volume
   tags:
     - glustervolume
+
+- name: Mounting volume on client
+  hosts: clients
+  gather_facts: false
+  roles:
+    - prepare-mount-client
+
+- name: export variables
+  hosts: clients
+  gather_facts: false
+  roles:
+    - export-env-variable
+
+- name: run smallfile
```

Comment:
Decision to run a test should come from the BT yml file, which we have not defined yet. This helps in deciding which test to run and hence invoke that test in particular.

I see bench-test-template.yml as follows:
```
benchmarks:
  smallfile:
    operation: [create|read|ls-l|...]
    threads: <n>
    filesize: <n>
    files: <n>
    top: <dir>
    remote-pgm-dir: <dir>
    host-set: <clients>
    output-json: <bool>
  smallfile:
    <next test>
  ...
  fio:
    jobfile: <filename> [see https://media.readthedocs.org/pdf/fio/latest/fio.pdf]
  ...
```